src/components/Powers.jsx

```diff
@@ -38,20 +38,19 @@ const Powers = (props) => {
     {
       id: 'power-lower-15kw',
       icon: <BoltIcon sx={iconRequirements} />,
-      textHeader: t('GURB_POWER_LOWER_15_HEADER')
+      textHeader: t('POWER_LOWER_15_HEADER')
     },
     {
       id: 'power-higher-15kw',
       icon: <BoltIcon sx={iconRequirements} />,
-      textHeader: t('GURB_POWER_HIGHER_15_HEADER')
+      textHeader: t('POWER_HIGHER_15_HEADER')
     }
   ]
 
-  console.log('values', values)
   return (
     <Grid container spacing={2}>
       <Grid item xs={12}>
-        <Typography sx={textHeader5}>{t('GURB_POWER_HELPER')}</Typography>
+        <Typography sx={textHeader5}>{t('POWER_HELPER')}</Typography>
```

Comment:
Same here! I'm not going to point all cases, but if it helps, tell me! :pancakes: